packages/exhibitor/buildinfo.json

```diff
@@ -9,8 +9,8 @@
     },
     "zookeeper": {
       "kind": "url_extract",
-      "url": "https://downloads.mesosphere.com/zookeeper/zookeeper-3.4.8.tar.gz",
-      "sha1": "51b61611a329294f75aed82f3a4517a4b6ff116f"
+      "url": "https://downloads.mesosphere.com/zookeeper/zookeeper-3.4.9.tar.gz",
```

Comment:
Also need to update the mesos bundled zookeeper to 3.4.9 if we want to do this. Want to have one version of zookeeper and zookeeper clients if at all possible.